src/types.rs

```diff
@@ -336,6 +336,10 @@ mod port_range_format {
 	}
 }
 
+fn default_libp2p_psk_path() -> String {
```

Comment:
wdyt of moving defaults to defaults module and then use it as `defaults::libp2p_psk_path`?